copyparty/web/browser.js

```diff
@@ -1878,9 +1878,636 @@ var Ls = {
 
 		"lang_set": "刷新以使更改生效？",
 	},
+	"deu": {
+		"tt": "Deutsch",
+
+		"cols": {
+			"c": "action buttons",
+			"dur": "Länge",
+			"q": "Qualität / Bitrate",
+			"Ac": "Audio Codec",
+			"Vc": "Video Codec",
+			"Fmt": "Format / Container",
+			"Ahash": "Audio Checksumme",
+			"Vhash": "Video Checksumme",
+			"Res": "Auflösung",
+			"T": "Dateityp",
+			"aq": "Audio Qualität / bitrate",
+			"vq": "Video Qualität / bitrate",
+			"pixfmt": "Subsampling / Pixelstruktur",
+			"resw": "horizontale Auflösung",
+			"resh": "vertikale Auflösung",
+			"chs": "Audio Channels",
+			"hz": "Sample Rate"
+		},
+
+		"hks": [
+			[
+				"misc",
+				["ESC", "close various things"],
+
+				"file-manager",
+				["G", "toggle list / grid view"],
+				["T", "toggle thumbnails / icons"],
+				["⇧ A/D", "thumbnail size"],
+				["ctrl-K", "delete selected"],
+				["ctrl-X", "cut selection to clipboard"],
+				["ctrl-C", "copy selection to clipboard"],
+				["ctrl-V", "paste (move/copy) here"],
+				["Y", "download selected"],
+				["F2", "rename selected"],
+
+				"file-list-sel",
+				["space", "toggle file selection"],
+				["↑/↓", "move selection cursor"],
+				["ctrl ↑/↓", "move cursor and viewport"],
+				["⇧ ↑/↓", "select prev/next file"],
+				["ctrl-A", "select all files / folders"],
+			], [
+				"navigation",
+				["B", "toggle breadcrumbs / navpane"],
+				["I/K", "prev/next folder"],
+				["M", "parent folder (or unexpand current)"],
+				["V", "toggle folders / textfiles in navpane"],
+				["A/D", "navpane size"],
+			], [
+				"audio-player",
+				["J/L", "prev/next song"],
+				["U/O", "skip 10sec back/fwd"],
+				["0..9", "jump to 0%..90%"],
+				["P", "play/pause (also initiates)"],
+				["S", "select playing song"],
+				["Y", "download song"],
+			], [
+				"image-viewer",
+				["J/L, ←/→", "prev/next pic"],
+				["Home/End", "first/last pic"],
+				["F", "fullscreen"],
+				["R", "rotate clockwise"],
+				["⇧ R", "rotate ccw"],
+				["S", "select pic"],
+				["Y", "download pic"],
+			], [
+				"video-player",
+				["U/O", "skip 10sec back/fwd"],
+				["P/K/Space", "play/pause"],
+				["C", "continue playing next"],
+				["V", "loop"],
+				["M", "mute"],
+				["[ and ]", "set loop interval"],
+			], [
+				"textfile-viewer",
+				["I/K", "prev/next file"],
+				["M", "close textfile"],
+				["E", "edit textfile"],
+				["S", "select file (for cut/copy/rename)"],
+			]
+		],
+
+		"m_ok": "OK",
+		"m_ng": "Abbrechen",
+
+		"enable": "Aktivieren",
+		"danger": "ACHTUNG",
+		"clipped": "in Zwischenablage kopiert",
+
+		"ht_s1": "Sekunde",
+		"ht_s2": "Sekunden",
+		"ht_m1": "Minute",
+		"ht_m2": "Minuten",
+		"ht_h1": "Stunde",
+		"ht_h2": "Stunden",
+		"ht_d1": "Tag",
+		"ht_d2": "Tage",
+		"ht_and": " und ",
+
+		"goh": "control-panel",
+		"gop": 'previous sibling">prev',
+		"gou": 'parent folder">up',
+		"gon": 'next folder">next',
+		"logout": "Abmelden ",
+		"access": " Zugriff",
+		"ot_close": "Submenu schliessen",
```

Comment:
`schließen`, or is this a de-ch translation?

Reply:
I follow the Swiss orthography, which omits `ß` for `ss` (<personal opinion>`ß` is unnecessary</personal opinion>)